src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicDoNotUseCountWhenAnyCanBeUsedFixer.vb

```diff
@@ -115,36 +108,41 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
 
             End While
 
-            Dim invocationExpression As InvocationExpressionSyntax = Nothing
-
             If isAsync Then
 
                 Dim awaitExpressionSyntax = TryCast(sourceExpression, AwaitExpressionSyntax)
 
                 If Not awaitExpressionSyntax Is Nothing Then
```

Comment:
Nit:
```suggestion
                If awaitExpressionSyntax IsNot Nothing Then
```

Reply:
These are already "IsNot" in master, They was fixed after I created this branch.